tests/integration/actions/config/base.py

```diff
@@ -149,6 +149,7 @@ def test(
 
         if not any((step.present, step.absent)):
             expected_output = retrieve_fixture_for_step(request, step.step_index)
-            assert expected_output == received_output, "\n" + "\n".join(
-                difflib.unified_diff(expected_output, received_output, "expected", "received"),
-            )
+            if expected_output:
```

Comment:
this change will not be necessary b/c the output key will always be present and only used it there isn't a present or absent key.

Without present, absent or expected_output, the test will alwasy blindly pass.

Reply:
Okey, removed.